xof/blake/blake.go

```diff
@@ -0,0 +1,82 @@
+// Package blake provides an implementation of kyber.XOF based on the
+// Blake2bx construction.
+package blake
+
+import (
+	"github.com/dedis/kyber"
+	"golang.org/x/crypto/blake2b"
+)
+
+type xof struct {
+	impl blake2b.XOF
+	key  []byte
+}
+
+// New creates a new XOF using the Blake2b hash.
+func New(seed []byte) kyber.XOF {
+	seed1 := seed
+	var seed2 []byte
+	if len(seed) > blake2b.Size {
+		seed1 = seed[0:blake2b.Size]
+		seed2 = seed[blake2b.Size:]
+	}
+	b, err := blake2b.NewXOF(blake2b.OutputLengthUnknown, seed1)
+	if err != nil {
+		panic("blake2b.NewXOF should not return error: " + err.Error())
+	}
+
+	if seed2 != nil {
+		_, err := b.Write(seed2)
+		if err != nil {
+			panic("blake2b.XOF.Write should not return error: " + err.Error())
+		}
+	}
+	return &xof{impl: b}
+}
+
+func (x *xof) KeySize() int {
```

Comment:
No need for comments because it implements the interface?

Reply:
yes.